src/components/ArticleList.jsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../api";
+
+import ArticleCard from "./ArticleCard";
+
+import "../css/ArticleCard.css";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+  const [isLoading, setIsLoading] = useState(false);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getArticles()
+      .then((articles) => {
+        setArticles(articles);
+        setIsLoading(false);
+        if (articles.length === 0) {
+        }
+      })
+      .catch(() => {
+        setIsLoading(false);
+      });
+  }, []);
+
+  if (isLoading) {
+    return <div role="alert">Loading Articles...</div>;
+  }
+
+  // console.log(articles);
```

Comment:
Dont forget to remove console.log's and un-needed comments before pushing your work up to github